Sources/ReleaseSubscriptions/App.swift

```diff
@@ -41,3 +54,9 @@ struct App: AsyncParsableCommand {
         }
     }
 }
+
+extension Logger {
+    fileprivate static var app: Logger {
```

Comment:
ワイは extension に `private` を付けることが多いけど、同じ意味かな？
どっちのほうが可読性が高いかな？

```suggestion
private extension Logger {
    static var app: Logger {
```

Reply:
同じ意味になります

今回はすでに存在しているコードたちは、`extension` の頭ではなく各プロパティ・メソッドの頭にアクセス修飾子に突いているのでそれに倣っています